packages/meditrak-server/src/apiV2/export/exportSurveyResponses/exportResponsesToFile.js

```diff
@@ -0,0 +1,303 @@
+/**
```

Comment:
Most of this comes from `exportSurveyResponses.js`, but with things moved around to be 
- more efficient (large exports are significantly faster, anecdotally)
- broken up into more discrete chunks
- able to be called from a place that doesn't extract the parameters from a url query (not actually required, but this is where I found the theoretical split in responsibilities between this file and the original)
- able to save multiple xlsx files and then zip them at the end, removing overhead from memory as it goes along (the main motivation!)